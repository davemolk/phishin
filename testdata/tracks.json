{"success":true,"total_entries":35069,"total_pages":1754,"page":1,"data":[{"id":4270,"show_id":217,"show_date":"1994-10-07","venue_name":"Stabler Arena, Lehigh University","venue_location":"Bethlehem, PA","title":"Maze","position":10,"duration":673672,"jam_starts_at_second":null,"set":"2","set_name":"Set 2","likes_count":4,"slug":"maze","tags":[],"mp3":"https://phish.in/audio/000/004/270/4270.mp3","waveform_image":"https://phish.in/audio/000/004/270/waveform-4270.png","song_ids":[486],"updated_at":"2023-10-27T22:30:27Z"},{"id":6693,"show_id":323,"show_date":"1993-04-09","venue_name":"State Theatre","venue_location":"Minneapolis, MN","title":"Stash","position":4,"duration":675971,"jam_starts_at_second":null,"set":"1","set_name":"Set 1","likes_count":1,"slug":"stash","tags":[{"id":1,"name":"SBD","priority":1,"group":"Audio","color":"#888888","notes":null,"transcript":null,"starts_at_second":null,"ends_at_second":null},{"id":4,"name":"Jamcharts","priority":4,"group":"Curated Selections","color":"#888888","notes":"Several minutes of growly, percussive, dissonant, and atypical jamming.","transcript":null,"starts_at_second":null,"ends_at_second":null}],"mp3":"https://phish.in/audio/000/006/693/6693.mp3","waveform_image":"https://phish.in/audio/000/006/693/waveform-6693.png","song_ids":[728],"updated_at":"2023-10-27T22:31:08Z"}]}