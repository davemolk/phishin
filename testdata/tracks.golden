ID:   Date:       Venue:                            Location:        Title:  Mp3:
4270  1994-10-07  Stabler Arena, Lehigh University  Bethlehem, PA    Maze    https://phish.in/audio/000/004/270/4270.mp3
6693  1993-04-09  State Theatre                     Minneapolis, MN  Stash   https://phish.in/audio/000/006/693/6693.mp3

Total Entries: 35069  Total Pages: 1754  Result Page: 1
